### PR TITLE
add ratio of disk usage to df widget.

### PR DESCRIPTION
**Pull request by himaaaatti**
This request makes df widget be like image.
disk

### CODE REVIEW

File: libqtile/widget/df.py [hunk 1]
```diff
@@ -34,9 +34,9 @@ class DF(base.ThreadedPollText):
         ('warn_space', 2, 'Warning space in scale defined by the ``measure`` option.'),
         ('visible_on_warn', True, 'Only display if warning'),
         ('measure', "G", "Measurement (G, M, B)"),
-        ('format', '{p} ({uf}{m})',
+        ('format', '{p} ({uf}{m}|{r:.0f}%)',
             'String format (p: partition, s: size, '
-            'f: free space, uf: user free space, m: measure)'),
+            'f: free space, uf: user free space, m: measure) r: ratio (uf/s)'),
```
Comment (flacjacket) [inline libqtile/widget/df.py]: The close paren after `measure` should be the furthest right

File: libqtile/widget/df.py [hunk 2]
```diff
@@ -68,6 +68,7 @@ def poll(self):
             text = ""
         else:
             text = self.format.format(p=self.partition, s=size, f=free,
-                    uf=self.user_free, m=self.measure)
+                    uf=self.user_free, m=self.measure,
+                    r=(size - self.user_free) / size * 100)
```
Comment (flacjacket) [inline libqtile/widget/df.py]: You should `from __future__ import division` so this always works ok.

**Comment by himaaaatti** [comment]
I fixed those!

**Comment by flacjacket** [comment]
Thanks for the fixes!

**Pull request closed without merging**